backend/docs/developers/setup-local-env.rst

```diff
@@ -43,22 +45,58 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
-#. Create a superuser to access the management interface: ``src/manage.py createsuperuser``
-#. To set environment variables settings, vreate a ``.env`` file. You can use and modify the provided example:
 
-  .. code:: bash
+   .. note::
 
-    cp dotenv.example .env
+      If you get an error about the ``django.contrib.gis`` module,
+      you need to install the GDAL library and/or enable the extension or give superuser privileges. See the requirements in the Dockerfile.
+
+#. Create a superuser to access the management interface:
+
+   .. code:: bash
+
+      src/manage.py createsuperuser
+
+#. To set environment variables settings, create a ``.env`` file. You can use and modify the provided example:
+
+   .. code:: bash
+
+      cp dotenv.example .env
+
+#. Generate the translation files:
+
+   .. code:: bash
+
+      ./bin/make_translations
+      src/manage.py makemessages -a
+      src/manage.py compilemessages --locale nl
 
 #. Run the development server with ``src/manage.py runserver``
 
+#. Optionally, you can load initial zaak data to populate your database.
 
-Optionally, you can load fixtures for the templates of the admin and for the admin index configuration:
+   .. note::
 
-.. code:: bash
+      This is a time-consuming process:
+
+      - ~30–60 minutes **without** a local Open Zaak instance
+      - ~1–2 minutes **with** a local Open Zaak instance
+
+   .. important::
+
+      For this to work, the services need to be set up correctly. You can either do this manually, or follow the instructions to :ref:`start a local Open Zaak instance <open-zaak-section>`.
+
+   To load the data, run:
 
-    src/manage.py loaddata fixture_name.json
+   .. code:: bash
 
+      src/manage.py cache_zaken
+
+#. Optionally, you can load fixtures for the templates of the admin and for the admin index configuration:
```

Comment:
This is a bit vague, you may want to say the "email templates" + the name of the fixture (and the other fixtures that should be loaded). A django developer would know how to load a fixture, so it is more useful to specify which fixtures are required.